crates/ingress-http/src/rpc_request_dispatcher.rs

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2024 -  Restate Software, Inc., Restate GmbH.
+// Copyright (c) 2024-2024 - Restate Software, Inc., Restate GmbH.
```

Comment:
```suggestion
// Copyright (c) 2024-2024 = 0 - Restate Software, Inc., Restate GmbH.
```

Reply:
Good catch. Not sure what the license generator did here.